proxies/python/install.sh

```diff
@@ -1,6 +1,17 @@
-if [ "$PYTHON_SDK_VERSION" ]; then
+#!/bin/sh
+
+if [[ -n "$SDK_GITHUB_SHA" ]]; then
+    echo "Checking out python SDK at $SDK_GITHUB_SHA"
+    cd ..
+    git clone https://github.com/DevCycleHQ/python-server-sdk.git
+    cd python-server-sdk
+    git checkout $SDK_GITHUB_SHA
+    cd ..
+    pip3 install -e ../python-server-sdk/
```

Comment:
These paths don't seem quite right to me, but maybe I'm missing something.
We go up one directory, then clone the SDK repo and cd into it, then back up to the parent directory again, but then reference the `python-server-sdk` from the parent directory?

Reply:
yeah i see what you mean.  the second `cd ..` really doesn't make sense. Like if anything shouldn't it be trying to go back down into the app/ folder where the proxy code is running?

Reply:
I think it is supposed to be `cd ../app` to get back into the proxy directory

Reply:
ah, i think the pip install is global since this is run as root so the directory change is likely unnecessary.  See my latest change.